docs/_includes/projects/properties/defaultconverter/my-element.js

```diff
@@ -64,7 +64,7 @@ class MyElement extends LitElement {
     this.prop1 = randy.toString();
     this.prop2 = randy;
     this.prop3 = !myBool;
-    this.prop4 = Object.assign([], [...this.prop4], randy);
+    this.prop4 = Object.assign([...this.prop4, randy]);
```

Comment:
As per comments above by @keanulee 

```suggestion
    this.prop4 = [...this.prop4, randy];
```